### ISSUE TITLE
Complete disk images are corrupt

### ISSUE THREAD

**Issue by Chaemelion** [sean-obrien/rpotter#7]
Both of the zipped raspbian/OpenCV/rpotter disk images on raspberrypotter.net are corrupt and cannot be extracted. Is there any chance someone can upload a working copy?

**Comment by Chaemelion** [comment]
Well, I take that back. Downloading both files multiple times in various ways and extracting or zip-recovery using various tools all fail on a linux machine. It seems to extract just fine on windows, however. Perhaps it's related to the fact that the zipped folder containing the image file is named the same thing as the file, extension and all? Either way, I'll just use Windows...

**Comment by ShortCipher** [comment]
Hi @Chaemelion, I had the same issue (with the only file I see on raspberrypotter.net now). Most unzip tools found a problem with rpotter.dmg.zip and would not extract. However, the "Archive Utility" on OS X works, and extracts rpotter.dmg.  I was able to mount this image on OS X to see the "boot" and "RECOVERY" drives, but Etcher (the recommended SD burning utility) says there is an issue with this image (in both zipped and unzipped forms).

After much trial and error, I was able to use "Disk Utility" to convert the unzipped image to a compressed .dmg file.  Etcher is happy with this image.  Now, I just need to find a 16 GB microSD card laying around here...
